From ed0178e4dfe4dc78c426a639baa8c7a94cca4d06 Mon Sep 17 00:00:00 2001
From: Jake Crosby <jakecrosby@gmail.com>
Date: Tue, 29 May 2012 10:08:06 +1200
Subject: [PATCH] pass data via prompt

---
 app/snapr.js |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/app/snapr.js b/app/snapr.js
index bbcf7ca..30b65cc 100644
--- a/app/snapr.js
+++ b/app/snapr.js
@@ -733,7 +733,7 @@ function spinner_stop()
 function pass_data( url )
 {
     //console.debug("pass data: ", url)
-    window.location = url.replace(/\+/g, '%20');
+    prompt(url.replace(/\+/g, '%20'));
 }
 
 function upload_progress( data, datatype )
-- 
1.7.7.4

